app/views/topics/show.html.erb

```diff
@@ -73,6 +72,9 @@
       <meta itemprop='interactionCount' content='UserLikes:<%= post.like_count %>'>
       <meta itemprop='interactionCount' content='UserComments:<%= post.reply_count %>'>
       <meta itemprop='headline' content='<%= @topic_view.title %>'>
+      <div class='clearfix'>
+        <span class='post-likes'>Likes: <%= post.like_count %></span>
```

Comment:
had to add a translation in server.en.yml since substitution doesn't work with js here. client.en.yml has {{count}} which ruby doesn't substitute with count value since it expects %{count}

Reply:
It would have been possible to change the placeholder in `client.en.yml` to `%{count}`. That works for 99% of all client strings. But, it looks like `js.post.has_likes` isn't used anymore since https://github.com/discourse/discourse/commit/9bf87f36650370f015e34761d6568d5173931cd8. So I guess adding a new translation is fine. You could even delete the unused `js.post.has_likes` from `client.en.yml`

Reply:
`has_likes` has been translated in multiple languages so I would have to delete or update them in multiple files. So I'm not updating/deleting it in this commit. I can do it later in separate commit if needed. :bowing_man:

Reply:
Deleting it from the English locale file is enough. Our translation workflow will remove it from all other languages at a later time. And it's gone... https://github.com/discourse/discourse/commit/66aa871c2462fcf8658173fb45c1f928bbd36fa5